File: app/src/main/java/com/glmiyamoto/learningkotlin/java/Operator.java
package com.glmiyamoto.learningkotlin.java;

public class Operator {
    private void math(final int a, final int b) {
        int c = 0;
        c = a + b;


        int d = 0;
        d = a - b;
    }

    private void increment(int a) {
        a++;
        a--;
    }

    private void operators(final int a, final int b) {
        if (a > b) return;
        if (a >= b) return;
        if (a < b) return;
        if (a <= b) return;
        if (a == b) return;
        if (a != b) return;

        final int c = a | b;
        final int d = a & b;
        final int e = a ^ b;
    }
}

File: app/src/main/java/com/glmiyamoto/learningkotlin/kotlin/Operator.kt
package com.glmiyamoto.learningkotlin.kotlin

class Operator {
    private fun math(a: Int, b: Int) {
        var c = 0
        c = a + b
        c = a.plus(b)

        var d = 0
        d = a - b
        d = a.minus(b)
    }

    private fun increment(a: Int) {
        a.inc()
        a.dec()
    }

    private fun operators(a: Int, b: Int) {
        if (a > b) return
        if (a >= b) return
        if (a < b) return
        if (a <= b) return
        if (a == b) return
        if (a != b) return

        val c = a or b
        val d = a and b
        val e = a xor b
    }
}